tests/chart/test_auth_sidecar.py

```diff
@@ -125,7 +124,7 @@ def test_auth_sidecar_security_context_with_dag_server_enabled(self, kube_versio
             ],
         )
 
-        common_dagserver_sts_test_cases(docs, 1)
+        common_sts_test_cases(docs, 1, "release-name-dag-server", "StatefulSet")
```

Comment:
This applies to more than just `sts`
```suggestion
        common_pod_manager_test_cases(docs, 1, "release-name-dag-server", "StatefulSet")
```